lib/rubocop/cop/style/if_unless_modifier.rb

```diff
@@ -26,6 +26,7 @@ def on_if(node)
           return if if_else?(node)
           return if node.chained?
           return unless fit_within_line_as_modifier_form?(node)
+          return if node.descendants.any? { |d| d.type == :if }
```

Comment:
This looks wrong to me - this should apply only to cases where the body is just some form of an `if` expression (meaning you should have just a single descendent).